### DISCUSSION TITLE
Some problem while importing meshio

### DISCUSSION THREAD

**Discussion by losanc** [nschloe/meshio#1252] [General]
Hi,

I just have a small problem while using meshio. 

Some background:
I am trying to build an addon for blender with meshio, because it's very complicated for blender to use `pip install meshio`, so I am putting meshio in the same directory with my code, then load it with `from . import meshio`

Problem:
 in `src/meshio/hmf/_hmf.py:1`, it writes as `import meshio`, since meshio is not in the `sys.path`, so it failed when `from . import meshio`.

Suggestion:
I am suggesting to replace `import meshio` with `from .._mesh import Mesh`, also need to change line 47 `return meshio.Mesh(` to `return Mesh(` and line 56 `write(filename, meshio.Mesh(points, cells), **kwargs)` to `write(filename, Mesh(points, cells), **kwargs)`
reason:
1. it solves the problem I have mentioned, 
2. also in general, I personally think it's better to use `from .._mesh import Mesh` rather than use `import meshio` here
2. I have checked other part of code, e.g. ply, vtk , they also use `from .._mesh import Mesh`

**Comment by nschloe** [comment]
Should definitely be fixed. Will take care of this soon!

**Comment by losanc** [comment]
Thanks!
